### PR TITLE
feat(WMS): allow getCapabilities to be turned off

### PR DESCRIPTION
**Pull request by milesap**
## Description
Avoids a getCapabilities call by providing a `resourceInfo` config parameter if the WMS ramp config has a property called `suppressGetCapabilities` and is set to `true`.

## Testing
brain

## Documentation
inline

## Checklist

- [ ] ~~`gulp test` succeeds without warnings or errors~~
- [ ] release notes have been updated
- [x] all commit messages are descriptive and follow guidelines
- [x] PR targets the correct release version
- I will assign this PR to the primary reviewer

---
This change is [Reviewable](https://reviewable.io/reviews/fgpv-vpgf/geoapi/314)

### CODE REVIEW

**Comment by james-rae** [comment]
___
*[src/layer/layerRec/wmsRecord.js, line 51 at r2](https://reviewable.io/reviews/fgpv-vpgf/geoapi/314#-LHi6RdXEO4ZVWSv7vPa:-LHi6RdXEO4ZVWSv7vPb:butjuzy) ([raw file](https://github.com/fgpv-vpgf/geoapi/blob/1088996961c1a981ca68877fcf7197a9968bc1cf/src/layer/layerRec/wmsRecord.js#L51)):*
> ```JavaScript
> 
>         if (this.config.suppressGetCapabilities) {
>             cfg.customLayerParameters.resourceInfo = {
> ```

pretty sure this should be `cfg.resrouceInfo = {`

see https://developers.arcgis.com/javascript/3/jsapi/wmslayer-amd.html#wmslayer1

**Comment by james-rae** [comment]
___
*[src/layer/layerRec/wmsRecord.js, line 51 at r2](https://reviewable.io/reviews/fgpv-vpgf/geoapi/314#-LHi6RdXEO4ZVWSv7vPa:-LHi8F0618W7QwmTR5lH:b-t19p7y) ([raw file](https://github.com/fgpv-vpgf/geoapi/blob/1088996961c1a981ca68877fcf7197a9968bc1cf/src/layer/layerRec/wmsRecord.js#L51)):*
<details><summary><i>Previously, james-rae (James Rae) wrote…</i></summary><blockquote>

pretty sure this should be `cfg.resrouceInfo = {`

see https://developers.arcgis.com/javascript/3/jsapi/wmslayer-amd.html#wmslayer1
</blockquote></details>

^ spelled it wrong....`cfg.resourceInfo`

**Comment by james-rae** [comment]
___
*[src/layer/layerRec/wmsRecord.js, line 51 at r3](https://reviewable.io/reviews/fgpv-vpgf/geoapi/314#-LHiM8HO8Z59atcjEPh2:-LHiM8HO8Z59atcjEPh3:b6vw1u1) ([raw file](https://github.com/fgpv-vpgf/geoapi/blob/935c92fe905cf0251f76b1b0782e37b3d20be5d0/src/layer/layerRec/wmsRecord.js#L51)):*
> ```JavaScript
> 
>         if (this.config.suppressGetCapabilities) {
>             cfg.resourceInfo = {
> ```

add this line above the `layerInfos` line
`extent: new this._apiRef.Map.Extent(-141, 41, -52, 83.5, {wkid: 4326}), // TODO make this a parameter post-demo`

**Comment by james-rae** [comment]
___
*[src/layer/layerRec/wmsRecord.js, line 51 at r3](https://reviewable.io/reviews/fgpv-vpgf/geoapi/314#-LHiM8HO8Z59atcjEPh2:-LHiMHchCFQ0Sq3vAaVS:bgolhp6) ([raw file](https://github.com/fgpv-vpgf/geoapi/blob/935c92fe905cf0251f76b1b0782e37b3d20be5d0/src/layer/layerRec/wmsRecord.js#L51)):*
<details><summary><i>Previously, james-rae (James Rae) wrote…</i></summary><blockquote>

add this line above the `layerInfos` line
`extent: new this._apiRef.Map.Extent(-141, 41, -52, 83.5, {wkid: 4326}), // TODO make this a parameter post-demo`
</blockquote></details>

so basically `resourceInfo` now has two properties
